_pages/policies/travel-and-leave-policies/moving.md

```diff
@@ -80,7 +80,13 @@ If you’re changing regions, your pay will change. Look up your new pay [here](
 
 ## Change your address
 
-1. Change your home address on [Employee Express](https://www.employeeexpress.gov/) to notify most GSA operations of your new address.
+1. Change your home address on [HR Links] self-service portal (https://hrlinks.gsa.gov/) to notify most GSA operations of your new address.
```

Comment:
could you clarify at "most GSA operations, that it includes payroll and OPM systems?

Reply:
@anjunainaustin are you wanting to list who is notified by this change or who still needs to be notified after making this update?